docs/installation/presto-configuration.rst

```diff
@@ -0,0 +1,47 @@
+
+==================
+Configuring Presto
+==================
+
+Presto has many configuration parameters that can be modified to
+tweak performance or add/remove features. While Presto should work decently well out-of-the-box,
+you still may need to make some changes.
+
+For example, it is often necessary to change the default memory configuration
+based on your cluster's capacity. The default max memory for each Presto server is 16GB, but in
+an environment with less than 16GB available per node, you will have to adjust the settings
+to be lower. Similarly, if you have a lot of memory (say, 120GB/node), you may want to allocate more
+memory to Presto for better performance.
+
+In the second case, in order to update the max memory value to 60GB per node, 
+change the line in ``/etc/opt/prestoadmin/coordinator/jvm.config`` and
```

Comment:
Do you think that's necessary? The procedure is exactly the same. The latter example is more helpful because it's more relevant to real use cases; I can add a note about the first one after explaining memory updates.